app/schemas/tech.relaycorp.gateway.data.database.AppDatabase/1.json

```diff
@@ -119,12 +119,46 @@
           }
         ],
         "foreignKeys": []
+      },
+      {
+        "tableName": "ParcelAck",
```

Comment:
I think `ParcelCollection` would be a more accurate name, since these records will exist before any their ACKs are generated/sent. Also, in the future we're likely to store metadata about the collection, like the address of the peer gateway.